0003-interledger-protocol/0003-interledger-protocol.md

```diff
@@ -172,67 +131,66 @@ As with the [internet protocol](https://tools.ietf.org/html/rfc791#section-2.3),
 
 The interledger module translates interledger addresses to local ledger addresses. Connectors and local ledger interfaces are responsible for translating addresses into interledger routes and local routes, respectively.
 
-Addresses are hierarchically structured strings consisting of segments delimited by the period (`.`) character. In order to distinguish the present address format from future or alternative versions, the protocol prefix `ilp:` MUST be used:
+Addresses are hierarchically structured strings consisting of segments delimited by the period (`.`) character.
 
 ```
-ilp:us.bank1.bob
+g.us.bank1.bob
 ```
 
-Care must be taken in mapping interledger addresses to local ledger accounts. Examples of address mappings may be found in "Address Mappings" ((TODO)).
+More information about ILP addresses can be found in the [ILP Address Specification](../0015-ilp-addresses/).
+
+The mapping from addresses to local accounts on a ledger is defined by the ledger protocol.
 
 ### Connectors
 
 Connectors implement the interledger protocol to forward payments between ledgers. Connectors also implement the [Connector to Connector Protocol (CCP)](../0010-connector-to-connector-protocol/) to coordinate routing and other interledger control information.
 
-![Interledger is an overlay across ledgers](assets/ilp-ledger-relation.png)
-
 ## Specification
 
-### ILP Header Format
+### ILP Payment Format
 
-Here is a summary of the fields in the ILP header format:
+Here is a summary of the fields in the ILP payment packet format:
 
 | Field | Type | Short Description |
 |:--|:--|:--|
-| version | INTEGER(0..255) | ILP protocol version (currently `1`) |
-| destinationAddress | IlpAddress | Address corresponding to the destination account |
-| destinationAmount | IlpAmount | Amount the destination account should receive, denominated in the asset of the destination ledger |
-| condition | IlpCondition | Execution condition for the transfers
-| expiresAt | IlpTimestamp | Maximum expiry time of the last transfer that the recipient will accept |
-
-#### version
-
-    INTEGER(0..255)
+| amount | UInt64 | Amount the destination account should receive, denominated in the asset of the destination ledger |
+| account | Address | Address corresponding to the destination account |
```

Comment:
Why the change from 'address' to 'account'? Will this same change be made in the [IPR](https://github.com/interledger/rfcs/blob/master/0011-interledger-payment-request/0011-interledger-payment-request.md#json) format?